book/03-git-branching/sections/basic-branching-and-merging.asc

```diff
@@ -139,38 +139,30 @@ index.html |    1 +
 1 file changed, 1 insertion(+)
 ----
 
-This looks a bit different than the `hotfix` merge you did earlier.
-In this case, your development history has diverged from some older point.
-Because the commit on the branch you're on isn't a direct ancestor of the branch you're merging in, Git has to do some work.
-In this case, Git does a simple three-way merge, using the two snapshots pointed to by the branch tips and the common ancestor of the two.
+Ito ay mukhang medyo iba sa `hotfix` na merge na ginawa mo kamakailan lamang. Sa kasong ito, ang kasaysayan ng iyong pag-develop ay humiwalay mula sa ilang mas matandang punto. Dahil ang commit sa branch kung saan nandoon ka ay hindi isang direktang ninuno ng branch na iyong mine-merge, ang Git ay dapat gumawa ng ilang trabaho. Sa kasong ito, ang Git ay gagawa ng isang simpleng three-way na merge, gamit ang dalawang snapshot na itinuro ng mga dulo ng branch at ang karaniwang ninuno ng dalawa.
```

Comment:
I think you should change mine-merge   to mini-merge

Reply:
Will update based on your suggestion. Thank you.